engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -2402,6 +2401,7 @@ public void execute(String[] args, ConsoleReader reader) throws Exception
           LOG.warn("Cannot flush command history");
         }
       }
+      yarnClient.stop();
```

Comment:
No, I don't feel strong, but can this be documented in the code? It is not clear why it is OK to use stop() in one place and close() in another without looking into YarnClient doc or source code.